README.md

```diff
@@ -122,10 +122,24 @@ In order to run integration tests against your own zendesk instance use the Cake
 .\build.ps1 --target=Run-Integration-Tests --zendeskUrl="<your zendesk url>" --zendeskUsername="<your zendesk username>" --zendeskToken="<your zendesk token>"
 ```
 
+### Integration Tests with Visual Studio
```

Comment:
the intent of this section might be better portrayed by "Running integrations tests on Visual Studio for Mac"

if we make this change then we can get rid of the below line as well as it is implicit